plugins/scaffolder-react/src/next/components/Stepper/Stepper.tsx

```diff
@@ -177,13 +185,13 @@ export const Stepper = (props: StepperProps) => {
                 Back
               </Button>
               <Button variant="contained" color="primary" type="submit">
-                {activeStep === steps.length - 1 ? 'Review' : 'Next'}
+                {activeStep === steps.length - 1 ? reviewButtonText : 'Next'}
```

Comment:
/cc @punkle maybe an alternative to storing at a `template.yaml`. Need to think about your PR a little more.